ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -0,0 +1,136 @@
+chain:
+  as: hypershift-kubevirt-create
+  steps:
+  - as: create-hostedcluster
+    cli: latest
+    env:
+    - name: HYPERSHIFT_NODE_COUNT
+      default: "3"
+      documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_BASE_DOMAIN
+      default: "ci.azure.devcluster.openshift.com"
+      documentation: "The cluster's FQDN will be a subdomain of the base domain."
+    commands: |-
+      set -exuo pipefail
+
+      # Deploy HyperConverged custom resource to complete kubevirt's installation
+      oc create -f - <<EOF
+      apiVersion: hco.kubevirt.io/v1beta1
+      kind: HyperConverged
+      metadata:
+        name: kubevirt-hyperconverged
+        namespace: openshift-cnv
+      EOF
+      
+      oc wait HyperConverged -n openshift-cnv kubevirt-hyperconverged --for=condition=Available --timeout=15m           
+
+      KUBEVIRT_CLUSTER_NAME="kv-nested"
+      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|cut -d- -f-2)"
+
+      echo "$(date) Creating HyperShift cluster ${KUBEVIRT_CLUSTER_NAME}"
+      /usr/bin/hypershift create cluster kubevirt \
+        --name ${KUBEVIRT_CLUSTER_NAME} \
+        --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
+        --base-domain ${HYPERSHIFT_BASE_DOMAIN} \
+        --pull-secret=/etc/ci-pull-credentials/.dockerconfigjson \
+        --release-image ${RELEASE_IMAGE_LATEST} \
+
+      echo "Waiting for cluster to become available"
+      oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${KUBEVIRT_CLUSTER_NAME}
+      echo "Cluster became available, creating kubeconfig"
+      bin/hypershift create kubeconfig --namespace=clusters --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
+
+      # Enable wildcard routes on the management cluster
+      oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p \
+        '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
+
+      export KUBECONFIG=${SHARED_DIR}/nested_kubeconfig
+
+      # Waiting for the default NodePort service to be created on the hosted cluster
+      until \
+        oc get service router-nodeport-default -n openshift-ingress
+      do
+        echo "router-nodeport-default service wasn't created yet on the hosted cluster."
+        sleep 10s
+      done
+
+      HTTPS_NODEPORT=$(oc get services -n openshift-ingress router-nodeport-default -o wide | awk '{print $5}' | awk -F "443:" '{print $2}' | awk -F "/" '{print $1}' | tr -d '[:space:]')
+      NESTED_BASE_DOMAIN=$(oc get ingresscontroller -n openshift-ingress-operator default -o yaml | grep domain | awk '{print $2 }')
```

Comment:
Ah, you get this domain already. So use this in the `--base-domain` cli arg.

Reply:
Done, thanks.